Makefile

```diff
@@ -2,7 +2,7 @@ CWD = $(shell pwd)
 SRC_DIR = ${CWD}/pootle
 DOCS_DIR = ${CWD}/docs
 STATIC_DIR = ${SRC_DIR}/static
-ASSETS_DIR = $(shell python -c "from pootle.settings import *; print(STATIC_ROOT)")
+ASSETS_DIR = ${SRC_DIR}/assets
```

Comment:
To reiterate what I said, from the app's standpoint a reference to a makefile from within the app's settings is confusing and makes no sense to me, even more when it refers to [a standard Django setting](https://docs.djangoproject.com/en/1.10/ref/settings/#std:setting-STATIC_ROOT). Besides that, no server admin is supposed to deal with the _10-base.conf_ file at all.

Moreover, the suggested approach for deploying Python projects is to install them as Python packages and as such no one should have to deal with build steps or setting up paths (among other things). Assets should be generated at package build time and hence the makefile is optional for deployment. Someone installing the app shouldn't be bothered with running any make steps in the first place (in fact, I'd be happy if our deployment didn't do so, but that's a different matter).